bin/cloudnet/tosca/network_diagrams.py

```diff
@@ -27,9 +27,13 @@
     # Target directory where network diagrams are generated.
     Generator.TARGET_DIRECTORY: "Results/NetworkDiagrams",
     # Network capability types.
-    "linkable_capability_types": ["tosca.capabilities.network.Linkable"],
+    "linkable_capability_types": ["tosca.capabilities.nfv.VirtualLinkable"],
     # Port capability types.
-    "bindable_capability_types": ["tosca.capabilities.network.Bindable"],
+    "bindable_capability_types": [
+        "tosca.capabilities.nfv.VirtualBindable",
+        "tosca.capabilities.nfv.TrunkBindable",
+        "tosca.capabilities.nfv.VipBindable",
+    ],
```

Comment:
Same remark. Previous value shall not be removed else network diagram generation for any TOSCA 1.2 template could fail. SOL001 specific types must be configured into the tosca2cloudnet.yaml file of SOL001-related examples.

Reply:
Currently, a default configuration file is loaded if exists. This file is tosca2cloudnet.yaml. Adding a command-line option to set the configuration file to load would be a great idea.